packages/frontend/.prettierrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  ...require('../../.prettierrc.json'),
+}
```

Comment:
_See comment on `packages/hardhat/.prettierrc.js`_

Reply:
you are right. removed it in [a582afe](https://github.com/yieldgate/yieldgate/pull/29/commits/a582afe1400d71e4338fe0fcace8ab8ee72b9333)